requirements.txt

```diff
@@ -1,3 +1,6 @@
+pytest==6.2.5
+pytest-xdist==2.5.0
```

Comment:
Why would we force all users to install dev dependencies? (and old ones at that)?

pytest is required to run tests - and is included (and pinned) in requirements-dev.txt.